src/payment/offsite-payment-mappers/payload-mapper.js

```diff
@@ -87,6 +87,10 @@ export default class PayloadMapper {
             this.storeMapper.mapToStore(data)
         );
 
+        if (payment.formattedPayload) {
```

Comment:
has bigpay been modified to read `formattedPayload` attribute? from what I can see here https://github.com/bigcommerce/bigpay/pull/2054/files, this might need to be called `info_3ds2` ?

IMO, something like `info_3ds2` makes more sense. This data is specific to 3ds2 so maybe the field should contain `3ds2`

Reply:
Every field contained in `formattedPayload` is sent separately (that's the reason `objectAssign` is used) so bigpay  has no need to read the formattedPayload object, instead it reads every field individually and it maintains the consistency with all the fields that are already sent by the mapper. 

I decided to use `formattedPayload` as used in the `payload-mapper` for online payment methods  as it seems a more 'general use' field not having to be tied to a specific functionality or data structure.

Also the field renamed in bigpay seems to be for the mapping when the provider sends the data, not when receiving it, so it should have no impact on this mapper.